### ISSUE TITLE
Different behaviour when clicking to fullscreen vs using the button/alt+enter

### ISSUE THREAD

**Issue by kuzyn** [silverwind/droppy#379]
There seems to be a different behaviour when playing a video and triggering full-screen mode:

- If you double click on the image, the video will go to full-screen and the cursor will disappear
- If you use the keyboard shortcut (f, alt-enter) OR use the top-right icon, the video will go to full screen and the cursor will not disappear

I looked a bit into it and it seems that the event generated by these action are different: clicking the video pass-through to Plyr, while the toggle doesn't seem to do the same thing. Any pointer on where to look to PR this? I couldn't seem to wrap my head around what `screenfull` was and where is the actual invocation for top-right the button press is.

Thanks!

**Comment by silverwind** [comment]
There are two fullscreen modes:

1. The Alt-Enter and button triggers [screenfull.js](https://github.com/sindresorhus/screenfull.js) which basically just puts the whole page in fullscreen (but still allows switching to next media etc.).
2. The plyr fullscreen which puts the `<video>` itself into fullscreen and which does cursor hiding but not allow to switch to other media.

I think cursor hiding is generally desireable in fullscreen (also on other content like images or pdf). Ideally we could detect if plyr wants to go fullscreen, prevent it, and trigger fullscreen via `screenfull` instead. Would need to replicate mouse hiding ourselves, thought.

On a sidenote, I fixed a bug with `screenfull`-induced fullscreen cutting off a section from the buttom of the video.

**Comment by kuzyn** [comment]
Yeah I agree that having a constant fullscreen behaviour makes logical sense. Hopefully I can queue this up before the summer end ;) :+1:
